code/modules/organs/wound.dm

```diff
@@ -276,7 +300,43 @@ datum/wound/cut/massive
 	stages = list("massive wound" = 70, "massive healing wound" = 50, "massive blood soaked clot" = 25, "massive angry scar" = 10,  "massive jagged scar" = 0)
 	damage_type = CUT
 
+/** PUNCTURES **/
+/datum/wound/puncture/can_worsen(damage_type, damage)
+	return 0
+/datum/wound/puncture/can_merge(var/datum/wound/other)
```

Comment:
E.g. instead of "he has a puncture wound and a puncture wound and a puncture wound on his chest" you'll get "he has several puncture wounds on his chest"